src/verifiable-credential.js

```diff
@@ -143,7 +178,12 @@ class VerifiableCredential {
    */
   addSubject(subject) {
     ensureObjectWithId(subject, 'credentialSubject');
-    this.subject = getUniqueElementsFromArray([...this.subject, subject], JSON.stringify);
+    if (!this.credentialSubject || this.credentialSubject.length === 0) {
+      this.credentialSubject = [subject];
+    }
+
+    const subjects = this.credentialSubject.length ? this.credentialSubject : [this.credentialSubject];
```

Comment:
This will make `subjects` array of empty array if `credential.credentialSubject` is empty which is not what you want

Reply:
good spot, thanks, resolved in vc class earlier too

Reply:
This hasn't changed

Reply:
the fix is above lines 181-183, `credential.credentialSubject` will never be empty

Reply:
I see. An easy alternative is `this.credentialSubject = getUniqueElementsFromArray([...this.credentialSubject, subject], JSON.stringify);` without above lines but its fine. Don't want to keep it blocked